blueocean-core-js/src/js/fetch.js

```diff
@@ -127,7 +145,7 @@ export const FetchFunctions = {
             const future = isoFetch(url, FetchFunctions.sameOriginFetchOption(fetchOptions))
                 .then(FetchFunctions.checkRefreshHeader)
                 .then(FetchFunctions.checkStatus)
-                .then(FetchFunctions.parseJSON);
+                .then(FetchFunctions.parseJSON, FetchFunctions.parseErrorJson);
```

Comment:
Why not a new line?

`
.then(FetchFunctions.parseJSON)
.then(FetchFunctions.parseErrorJson)
 `